data/sql_setup/prepare_schedule_c.sql

```diff
@@ -0,0 +1,22 @@
+-- Create trigger to maintain Schedule C search columns
+ALTER TABLE fec_vsum_sched_c DROP COLUMN if EXISTS candidate_name_text;
+ALTER TABLE fec_vsum_sched_c DROP COLUMN if EXISTS loan_source_name_text;
+
+ALTER TABLE fec_vsum_sched_c add column candidate_name_text tsvector;
+ALTER TABLE fec_vsum_sched_c add column loan_source_name_text tsvector;
+
+UPDATE fec_vsum_sched_c SET candidate_name_text = to_tsvector(cand_nm);
+UPDATE fec_vsum_sched_c SET loan_source_name_text = to_tsvector(loan_src_nm);
```

Comment:
These statements are needed to update all the vector columns with the input from their respective native column that already exists.  Note up above the columns are dropped if they already exist.  So essentially any time the prepare directory is ran through a manager task, these two columns are dropped and created, just for integrity's sake.  I do agree it's not needed, but if those two statements aren't here they should at least be commented out for the purposes of documentation, as someone will have to run these from the console.